packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -98,13 +135,22 @@ class AccountImportOptions extends Component {
       return;
     }
 
-    const chainId = parseInt(chainIdString);
+    const signerChainId = parseInt(chainIdString);
+
+    if (!this.isCurrentChainIdTheAddressForImportChainId(signerChainId)) {
```

Comment:
nitpick but the chainId isn't an information inside the address like the function name could suggest, it's more like a meta data that comes alongside with the address

maybe `currentChainId !== signerChainId` and remove the function? we don't have to use bignumber here afaik

Reply:
i've pushed commit de3396ff7da8eb4dca9bbac347ecf6f0f5e513c3 to address this

Reply:
i've addressed this issue in commit c4cf7d580da8e2926d78b2092760300afe2fedfd and de3396ff7da8eb4dca9bbac347ecf6f0f5e513c3